### PR TITLE
[HAL-1983] Update the Installer history screen to include the combina…

### PR DESCRIPTION
**Pull request by spyrkob**
…tion of manifest names/versions in the list

Issue: https://issues.redhat.com/browse/HAL-1983

### CODE REVIEW

**Comment by hpehl** [comment]
Thanks for the fix @spyrkob 
I'll review and merge asap.

**Comment by spyrkob** [comment]
@hpehl I also wanted to add the information on current channel versions to the standalone/domain attributes page (/console/index.html#standalone-server), but I couldn't figure out how to extend it. Is it possible to add this information to the ROOT_TEMPLATE metadata?

**Comment by hpehl** [comment]
@spyrkob Which page do you refer to? This one:
admin_admin@localhost_9933_console_index html

**Comment by spyrkob** [comment]
@hpehl yes, that's the one I was thinking of.

File: app/src/main/java/org/jboss/hal/client/installer/UpdateColumn.java [hunk 1]
```diff
@@ -158,7 +163,17 @@ public HTMLElement getIcon() {
 
             @Override
             public HTMLElement element() {
-                return ItemDisplay.withSubtitle(item.getName(), Format.mediumDateTime(item.getTimestamp()));
+                final HtmlContentBuilder<HTMLElement> builder = span()
```
Comment (hpehl) [inline app/src/main/java/org/jboss/hal/client/installer/UpdateColumn.java:166]: Please add the missing CSS class `itemText`
 https://github.com/hal/console/blob/3c853acc2d293826b55bcda34de2fbdb2b43fda2/core/src/main/java/org/jboss/hal/core/finder/ItemDisplay.java#L113

```java
span().css(itemText)
```

**Comment by hpehl** [comment]
I suggest filing another issue to include the channel version to the server/domain runtime page. 
What management operation must be used to get this information? Is that the adjusted `:product-info()` operation?

**Comment by spyrkob** [comment]
OK, created https://issues.redhat.com/browse/HAL-1986 to track the adding the current channel-versions.

>What management operation must be used to get this information? Is that the adjusted :product-info() operation?

Yes, or I can add a separate operation to get just the current channel versions, if it makes it easier to consume - added this info to the ticket as well

**Review by hpehl — APPROVED** [review]
LGTM! Thanks @spyrkob

**Comment by hpehl** [comment]
The `:product-info()` operation should work.

**Pull request merged**